### DISCUSSION TITLE
Installing primefaces theme in eclipse

### DISCUSSION THREAD

**Discussion by engegov** [primefaces/community#1660] [PrimeFaces] [answered]
Hi!

I need install a Primefaces Freya Theme in my Eclipse IDE and run in Tomcat7.  How could I import this theme? Is it necessary a .war file?

Please, need help.

Thanks in advance

**Comment by melloware** [comment, marked as answer]
Well i think you should start with this tutorial: https://balusc.omnifaces.org/2020/04/jsf-23-tutorial-with-eclipse-maven.html
